src/Blueprint/Blueprint/Models/PSLockMode.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Blueprint.Models
+{
+    public enum PSLockMode
+    {
+        Unknown = 0,
+        None = 1,
+        AllResources = 2
+    }
```

Comment:
These are from the old API version. We should use the new ones.

Reply:
Updated (They were updated at some point, must have discarded them somehow and didn't notice. Anyway, thanks for pointing out).

---
In reply to: [257120202](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257120202) [](ancestors = 257120202)